### DISCUSSION TITLE
LD2410 как сформулировать триггрер для правила?

### DISCUSSION THREAD

**Discussion by Bulatovor1** [arendst/Tasmota#20802] [Q&A]
LD2410 подключен к тасмота. В консоли выдает расстояние в виде LD2410":{"Distance":[30.0,57.0,0.0], Как из этого сформировать триггер для правил

**Comment by Jason2866** [comment]
This is an English only Discussion forum.

**Comment by joba-1** [comment]
Ваш вопрос не очень конкретный, поэтому я могу ответить только ссылками на примеры в документации. 

https://tasmota.github.io/docs/Rules/

https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached

И перестаньте убивать своих братьев. Вам нечего делать в Украине - езжайте домой. Мир!


Your question is not very specific, so I can only answer with links to examples in the documentation. 

https://tasmota.github.io/docs/Rules/

https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached

And stop killing your brothers. You have nothing to do in Ukraine - go home. Peace!
